stake-pool/cli/src/main.rs

```diff
@@ -826,6 +1107,91 @@ fn main() {
                     .help("Fee denominator, fee amount is numerator divided by denominator."),
             )
         )
+        .subcommand(SubCommand::with_name("create-vsa").about("Create a new validator stake account to use with the pool")
```

Comment:
`vsa` is a bit unclear for cli users, so could you expand it to `validator-stake` in the command names?